src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -121,9 +121,9 @@
     <value>General</value>
     <comment>Navigation view item name for General</comment>
   </data>
-  <data name="Shell_Espresso.Content" xml:space="preserve">
-    <value>Espresso</value>
-    <comment>Product name: Navigation view item name for Espresso</comment>
+  <data name="Shell_Awake.Content" xml:space="preserve">
+    <value>Awake</value>
+    <comment>Product name: Navigation view item name for Awake</comment>

```

Comment:
We also need to update `src\settings-ui\Microsoft.PowerToys.Settings.UI\loc` folder. Otherwise, localization isn't going to work.

@crutkas
Is it something that the localization team is going to do? Or should we do it?

Reply:
loc team is going to handle this so we should be fine.